src/use_case/recommend_word/RecommendOutputBoundary.java

```diff
@@ -0,0 +1,5 @@
+package use_case.recommend_word;
+
+public interface RecommendOutputBoundary {
+    void prepareRecommendView(RecommendOutputData recommendation);
```

Comment:
When naming try to be consistent since it gets confusing. For example here you initialized RecommendOutputData as "recommendation" but in the RecommendInteractor you initialized it as "recommendOutputData". Consistency in variable names can make the code easier to understand and maintain.